### DISCUSSION TITLE
Window Associations (autotype) does not pickup on applications.

### DISCUSSION THREAD

**Discussion by vasiousbloods** [keepassxreboot/keepassxc#6237] [Q&A] [answered]
when pressing my global auto-type hotkey, if there is a url or a link, its never failed to find the right login. 
For applications it just comes up with the error:
"Couldn't find an entry that matches the window title:"
I *have* added the right window in the correct login though >edit>auto-type>+>[window name].
it *does* see the window names, though. Just not acting on it.

I am using manjaro, and awesomeWM.

Any help would be greatly appreciated :)

**Comment by droidmonkey** [comment, marked as answer]
Try trimming the window name to the most consistent name and add a * to the beginning and end: `*Notepad*`

**Reply by vasiousbloods** [reply]
Thank you for your reply.
I tried with * before and after but the same error came up.
I tried all the combinations of names, also.

**Reply by droidmonkey** [reply]
You might be running into the problem I corrected recently with this PR: https://github.com/keepassxreboot/keepassxc/pull/6204

You can give this a try using a snapshot build at https://snapshot.keepassxc.org

**Reply by vasiousbloods** [reply]
aha okay! it works, but with a quirk. 
a little window pops up when I press the global hotkey. It did not recognize my window, but when I searched the popup, my login was not there! 
So i moved it to a different folder- to one where I saw some others appear in the pop up, and this time when i pressed the hotkey it recognized it :)
All this, 100% regardless of what autotype window i add to the login, it seems.
 Im going to go back to the main build- will it be patched soon?

**Reply by droidmonkey** [reply]
Ahhh so you may have had that group setup as Auto-Type disabled. If that is the case then there is nothing wrong with the current version. You also got to see the new Auto-Type selection window with database search. I hope to release the modal window fix with 2.6.5 in the coming weeks.
